runtime/src/runtime.js

```diff
@@ -5,50 +5,34 @@ import { ManagedMemory } from "./core/memory";
 import { GrainRunner } from "./core/runner";
 import { defaultFileLocator } from "./utils/locators";
 
-export let grainModule;
-
-// Workaround for this memory being initialized statically on module load
-export const memory = new WebAssembly.Memory({
-  initial: process.env.GRAIN_INIT_MEMORY_PAGES || 64,
-  maximum: process.env.GRAIN_MAX_MEMORY_PAGES,
```

Comment:
Can we get rid of these env vars then (including removing from setting in the CLI)?

Reply:
Should be all set here.